eng/common/pipelines/templates/steps/daily-dev-build-variable.yml

```diff
@@ -19,6 +19,7 @@ steps:
     if (('$(Build.Reason)' -in 'Schedule', 'IndividualCI') -and ('$(System.TeamProject)' -eq 'internal')) {
       $setDailyDevBuild = "true"
     }
-    echo "##vso[task.setvariable variable=SetDevVersion]$setDailyDevBuild"
+    echo "##vso[task.setvariable variable=SetDevVersion;isOutput=true]$setDailyDevBuild"
```

Comment:
Do you know if this will break any usages of this variable in the same job that reference it via `$(SetDevVersion)`? If so we either need to adjust those or set it both ways here.

Reply:
I think all ```isOutput=true``` does is allow the variable to be accessed outside the job/stage. My interpretation is based on his documentation:

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#set-a-multi-job-output-variable